vcluster/deploy/topologies/high-availability.mdx

```diff
@@ -27,10 +29,12 @@ controlPlane:
 ```
 
 ```
-vcluster create ha-tutorial --connect=false -f values.yaml
+vcluster create ha-tutorial --connect=false -f vcluster.yaml
 ```
 
-We’ve named the virtual cluster ha-tutorial. By default, the vcluster create command connects to the virtual cluster, but for the purposes of this tutorial, we’ve disabled that with the --connect=false flag. And we’ve specified the distro and the values.yaml file to use when creating the virtual cluster.
+We’ve named the virtual cluster ha-tutorial. By default, the `vcluster create` command connects to the virtual cluster,
+but for the purposes of this tutorial, we’ve disabled that with the `--connect=false flag`.
+We’ve also specified the distro and the vcluster.yaml file to use when creating the virtual cluster.
```

Comment:
Remove this line `We’ve also specified the distro and the vcluster.yaml file to use when creating the virtual cluster.` as it's more specific to the old way of deploying.